### DISCUSSION TITLE
Looking for help creating a stacked bar chart using a single sensor

### DISCUSSION THREAD

**Discussion by rwelsh09** [dbuezas/lovelace-plotly-graph-card#434] [Q&A]
I need help creating a graph like this.
![image](https://github.com/user-attachments/assets/95a8ce7a-d44f-46c5-8c45-44c4f95f5272)
These values all come from a single sensor.

The python used to create this is [here](https://github.com/babybuddy/babybuddy/blob/master/reports/graphs/pumping_amounts.py).

Thanks!

**Comment by dbuezas** [comment]
I suggest you look at the examples index and/or search for stacked bar chart in the discussion

**Reply by rwelsh09** [reply]
I have, the only stacked bar charts I could find used multiple sensors. 

I assume I'm going to need to use filters but I haven't figured out how to do so.

**Reply by dbuezas** [reply]
https://github.com/dbuezas/lovelace-plotly-graph-card/discussions/422

This one should help. Meta holds attributes. More in readme
